packages/ag-charts-community/src/chart/toolbar/toolbar.ts

```diff
@@ -622,7 +673,7 @@ export class Toolbar extends BaseModuleInstance implements ModuleInstance {
             if (this[group] == null) continue;
             const groupVisible = isGroupVisible(group);
             for (const button of this.groupButtons[group]) {
-                const buttonVisible = groupVisible && this[group].buttonConfigurations().some(isButtonVisible(button));
+                const buttonVisible = groupVisible && isButtonVisible(button);
```

Comment:
Did you mean to commit this?

You'll need to revert this change, the `isButtonVisible(button)` call returns a function so will always be truthy. And we need the `buttonConfigurations()` bit to fix issues with locale changes resetting buttons I believe (see https://github.com/ag-grid/ag-charts/commit/d4e833f3c766837ce7898e83eea3603aeda9d0f5).